pep-0590.rst

```diff
@@ -16,6 +16,9 @@ It introduces a new "vectorcall" protocol and calling convention.
 This is based on the "fastcall" convention, which is already used internally by CPython.
 The new features can be used by any user-defined extension class.
 
+Most new API is private in CPython 3.8, with plans to finalized and
+made public in Python 3.9.
```

Comment:
This is not grammatically correct. I would write
```
Most new API is private in CPython 3.8. The plan is to make it public in Python 3.9.
```

Reply:
"Most of the new API is private..." would be better.